upgrades/contexts.go

```diff
@@ -33,26 +34,32 @@ type Context interface {
 	// APIContext returns a new Context suitable for API-based upgrade
 	// steps.
 	APIContext() Context
+
+	//ServiceConfig
+	ServiceConfig() service.UpgradableService
```

Comment:
Now that you've changed the install step to construct a SystemdServiceManager itself, you should remove this.